Applications/Analyze/test/arm26.osim

```diff
@@ -683,7 +683,7 @@
 					<frames>
 						<PhysicalOffsetFrame name="ground_offset">
 							<!--Path to a Component that satisfies the Socket 'parent' of type C (description: The parent frame to this frame.).-->
-							<socket_parent_connectee_name>ground</socket_parent_connectee_name>
+							<socket_parent_connectee_name>../../../ground</socket_parent_connectee_name>
 							<!--Translational offset of this frame's origin from the parent frame's origin, expressed in the parent frame.-->
```

Comment:
@chrisdembia is this the end game/format or an intermediate, or older format used for testing, can you clarify? Thanks

Reply:
Also if the change is supposed to handle relative and absolute path, wouldn't it be backward compatible and not require any file changes? Sorry if you answer that later in the PR, it just hit me as I started looking at the diffs. Thank you